test/src/e2e_vm_tests/test_programs/should_pass/language/gtf_intrinsic/src/main.sw

```diff
@@ -0,0 +1,11 @@
+script;
+
+const SOME_TX_FIELD = 0x42;
+const SOME_OTHER_TX_FIELD = 0x77;
+
+fn main() -> u64 {
+    // Test expected to compile but revert because `fuell-core` does not support `gtf` yet.
+    let u64_field = __gtf::<u64>(1, SOME_TX_FIELD);
+    let b256_field = __gtf::<b256>(2, SOME_OTHER_TX_FIELD);
```

Comment:
I don't know what the `__gtf` op is supposed to do, but if the values of `u64_field` and `b256_field` are deterministic, it might be useful to add an assert to validate the values.

Reply:
Sadly, we can't do this just yet because `fuel-core` is not ready yet. So, the test will always revert at the `gtf` instruction before anything gets assigned.